src/test/java/demo/MySqlDbTest.java

```diff
@@ -0,0 +1,108 @@
+package demo;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashSet;
+import java.util.Set;
+
+public class MySqlDbTest {
+    public static final String MYSQL_URL = "jdbc:mysql://localhost:3306/test";
+    public static final String MYSQL_USER = "root";
+    public static final String PASSWORD = System.getenv("DB_PASSWORD");
```

Comment:
I'm just curious. Is `DB_PASSWORD` some default environment variable, created by MySQL? Or should it be set manually before running the tests?

Reply:
it should be set manually before running tests